web/styles/informational_overlays.css

```diff
@@ -73,8 +73,8 @@
 
     .arrow-key {
         font-size: 1.36em;
-        line-height: 1;
-        padding: 0 0.2em 0.2em;
+        line-height: normal;
+        padding: 0 0.2em;
```

Comment:
I don't understand what's being achieved here or in `app_components` with zeroing out the bottom padding. I'm seeing numerous keys that appear to be too low in their box, and removing the bottom padding seems like a likely culprit. And `normal` as a line-height value is much more difficult to reason about when trying to do precision alignment.

Reply:
![Screenshot 2025-02-19 at 4 29 39 PM](https://github.com/user-attachments/assets/6456ba5e-c5b6-49ad-b48a-9e27acd87627)